hw/ip/rstmgr/data/rstmgr.sv.tpl

```diff
@@ -298,7 +298,7 @@ module rstmgr
     .scanmode_i(prim_mubi_pkg::mubi4_test_true_strict(rst_ctrl_scanmode[0])),
     .scan_rst_ni,
     .rst_req_i(pwr_i.rst_sys_req | {PowerDomains{ndm_req_valid}}),
-    .rst_parent_ni(rst_lc_src_n),
+    .rst_parent_ni(rst_por_aon_n),
```

Comment:
don't we need to change this file to handle the reset reasons? since `ndm_reset` passes from `rv_dm` to `pwrmgr` now, we probably need to adjust the reset reset setting accordingly.

Reply:
Did you mean {PowerDomains{ndm_req_valid}} in line 300? I removed it because ndm_reset_req passes from rv_dm to pwrmgr as you pointed out. Please correct me if I am wrong.

Reply:
ah i meant here (https://github.com/lowRISC/opentitan/blob/2aedabb1e74628a7d7e4a503d8fbe63566af9def/hw/ip/rstmgr/data/rstmgr.sv.tpl#L418-L419)

In the old way, `ndm_rst_req` was handled in rstmgr directly. But how it's going through pwrmgr and coming back, so maybe we could update it the other info sources now instead of handling it differently.

Reply:
it would be good to have a look at the reset info and see what needs to be adjusted.
@matutem